test/e2e/pullrequest/ctrl.go

```diff
@@ -322,7 +322,7 @@ var _ = Describe("[e2e] Pull request controller", func() {
 		Expect(err).NotTo(HaveOccurred(), "Verify PullRequestTrigger error")
 
 		By("Verifying PullRequestQueue has been created and PullRequestTrigger has been deleted")
-		err = wait.PollImmediate(verifyTime1s, verifyTime75s, func() (ok bool, err error) {
+		err = wait.PollImmediate(verifyTime1s, verifyTime90s, func() (ok bool, err error) {
```

Comment:
should we try to update image tag https://github.com/agoda-com/samsahai/blob/f6e9f20d8a43320c829895ac55d4a7a36b666795/test/e2e/pullrequest/ctrl.go#L1273 to a newer version instead?

or maybe change to 
```
wordpressImage = s2hv1.ComponentImage{
		Repository: "bitnami/wordpress",
		Pattern:    "{{ .PRNumber }}",
	}
	wordpressImageTag = "latest"

	wordpressInvalidImage = s2hv1.ComponentImage{
		Repository: "invalid/wordpress",
		Pattern:    "invalid-{{ .PRNumber }}",
	}
	mariaDBImage = s2hv1.ComponentImage{
		Repository: "bitnami/mariadb",
		Pattern:    "{{ .PRNumber }}",
	}
	mariaDBImageTag = "latest"
```

and `PRNumber: latest`
I think the image tag might be very outdated